x-pack/plugins/ml/public/application/data_frame_analytics/pages/analytics_management/components/models_management/models_list.tsx

```diff
@@ -355,91 +382,51 @@ export const ModelsList: FC = () => {
     },
   ];
 
-  const pagination = {
-    initialPageIndex: pageIndex,
-    initialPageSize: pageSize,
-    totalItemCount: items.length,
-    pageSizeOptions: [10, 20, 50],
-    hidePerPageOptions: false,
-  };
+  const filters =
```

Comment:
This will let you get rid of ts-ignore 
```suggestion
  const filters: SearchFilterConfig[] =
```

Reply:
Good catch - updated in 5b160352ccb58bc3649269931f7917bd0316bceb